docs/content/guides/developer/sui-101/using-events.mdx

```diff
@@ -36,159 +30,220 @@ To create an event in your Move modules, add the `sui::event` dependency.
+| `Transaction`     | Events emitted from the specified transaction            | `{"Transaction":"DGUe2TXiJdN3FI6MH1FwghYbiHw+NKu8Nh579zdFtUk="}`                                      |
+| `MoveModule`      | Events emitted from the specified Move module            | `{"MoveModule":{"package":"<PACKAGE-ID>", "module":"nft"}}`                                           |
+| `MoveEventModule` | Events emitted, defined on the specified Move module.    | `{"MoveEventModule": {"package": "<DEFINING-PACKAGE-ID>", "module": "nft"}}`                          |
+| `MoveEvent`       | Move struct name of the event                            | `{"MoveEvent":"::nft::MintNFTEvent"}`                                                                 |
+| `EventType`       | Type of event described in Events section                | `{"EventType": "NewObject"}`                                                                          |
+| `Sender`          | Query by sender address                                  | `{"Sender":"0x008e9c621f4fdb210b873aab59a1e5bf32ddb1d33ee85eb069b348c234465106"}`                     |
+| `Recipient`       | Query by recipient                                       | `{"Recipient":{"AddressOwner":"0xa3c00467938b392a12355397bdd3d319cea5c9b8f4fc9c51b46b8e15a807f030"}}` |
+| `Object`          | Return events associated with the given object           | `{"Object":"0x727b37454ab13d5c1dbb22e8741bff72b145d1e660f71b275c01f24e7860e5e5"}`                     |
+| `TimeRange`       | Return events emitted in [start_time, end_time] interval | `{"TimeRange":{"startTime":1669039504014, "endTime":1669039604014}}`                                  |
 
-console.log(
-	await client.getObject({
-		id: Package,
-		options: { showPreviousTransaction: true },
-	}),
-);
+## Monitoring events
 
-let unsubscribe = await client.subscribeEvent({
-	filter: { Package },
-	onMessage: (event) => {
-		console.log('subscribeEvent', JSON.stringify(event, null, 2));
-	},
-});
-
-process.on('SIGINT', async () => {
-	console.log('Interrupted...');
-	if (unsubscribe) {
-		await unsubscribe();
-		unsubscribe = undefined;
-	}
-});
-```
+Firing events is not very useful in a vacuum. You also need the ability to respond to those events. There are to methods from which to choose when you need to monitor on-chain events:
+- Incorporate a [custom indexer](../advanced/custom-indexer.mdx) to take advantage of Sui's micro-data ingestion framework.
+- Poll the Sui network on a schedule to query events.
 
-#### Response
+Using a custom indexer provides a near-real time monitoring of events, so is most useful when your project requires immediate reaction to the firing of events. Polling the network is most useful when the events you're monitoring don't fire often or the need to act on those events are not immediate. The following section provides a polling example.
 
-When the subscribed to event fires, the example displays the following JSON representation of the event.
+## Examples
```

Comment:
Similarly, this section seems like it belongs near the other smaller examples about using GraphQL or `suix_queryEvents` via curl?